state-sync/diem-data-client/src/diemnet/mod.rs

```diff
@@ -44,7 +44,7 @@ mod tests;
 
 // TODO(philiphayes): configuration / pass as argument?
 const DEFAULT_TIMEOUT: Duration = Duration::from_millis(10_000);
-pub const DATA_SUMMARY_POLL_INTERVAL: Duration = Duration::from_millis(100);
+pub const DATA_SUMMARY_POLL_INTERVAL: Duration = Duration::from_millis(1_000);
```

Comment:
'cc @khiemngo -- one thing that would be good would be to add a new `DiemNetDataClient` config so we can make these constants configurable. We can then pass it to the `DiemNetDataClient::new(...)` function (where we already pass in the `StorageServiceConfig`). Let's sync on this 😄